src/main/java/banking/dao/AccountDAO.java

```diff
@@ -0,0 +1,155 @@
+    private final static String SELECT_ID = "SELECT id FROM account WHERE number=? AND pin=?";
+    private final static String GET_CARD = "SELECT * FROM account WHERE id = ?";
+    private final static String UPD_BALANCE = "UPDATE account SET balance = (balance + ?) WHERE id = ?";
+    private final static String GET_MONEY = "UPDATE account SET balance = (balance + ?) WHERE number = ?";
+    private final static String SEND_MONEY = "UPDATE account SET balance = (balance - ?) WHERE id = ?";
+    private final static String DEL_ACC = "DELETE FROM account WHERE id = ?";
+    private DBManager dbManager;
+
+    public AccountDAO(DBManager dbManager) {
+        this.dbManager = dbManager;
+    }
+
+    public void insertCard(Account account) {
+        try (
+             PreparedStatement preparedStatement = dbManager.getConnection().prepareStatement(INSERT_INTO)) {
+            preparedStatement.setString(1, account.getCard().getNumber());
+            preparedStatement.setString(2, account.getCard().getPIN());
+            preparedStatement.executeUpdate();
+        } catch (SQLException ex) {
+            ex.printStackTrace();
+        }
+    }
+
+    public int getId() {
+        int id = 0;
+        try (
+             Statement stmt = dbManager.getConnection().createStatement()) {
+
+            ResultSet res = stmt.executeQuery(SELECT_ALL);
+            while (res.next()) {
+                id = res.getInt("id");
+            }
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
+        }
+        return id;
+    }
+
+    public boolean containCard(String cardNumber) {
+        ResultSet res;
+        try (PreparedStatement preparedStatement = dbManager.getConnection().prepareStatement(SELECT_CARD_NUMBER)) {
+            preparedStatement.setString(1, cardNumber);
+            res = preparedStatement.executeQuery();
+            if (res.next()) {
+                return res.getBoolean(1);
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return false;
+    }
+
+    public int login(String cardNumber, String PIN) {
+        ResultSet res;
+        try (PreparedStatement preparedStatement = dbManager.getConnection().prepareStatement(SELECT_ID)) {
+            preparedStatement.setString(1, cardNumber);
+            preparedStatement.setString(2, PIN);
+            res = preparedStatement.executeQuery();
+            if (res.next()) {
+                return res.getInt("id");
+            } else {
+                return -1;
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return -1;
+    }
+
+    public int getBalance(int id) {
+        int balance = -1;
+        try (PreparedStatement preparedStatement = dbManager.getConnection().prepareStatement(GET_CARD)) {
+            preparedStatement.setInt(1, id);
+            ResultSet res = preparedStatement.executeQuery();
+            assert res != null;
```

Comment:
This check is redundant. Can you please remove it?

Reply:
fixed

Reply:
I can still see
```
assert res != null;
```

Reply:
fixed